utils/gql.py

```diff
@@ -43,6 +92,17 @@ def query(self, query, variables=None):
 
         result = json.loads(result_json)
 
+        # show schemas if log level is debug
+        for s in result['extensions']['schemas']:
+            logging.debug(['schema', s])
+
+        # we need to test self._valid_schemas to let through the query that
+        # retrieves the schemas in the __init__ method
+        if self.validate_schemas and self._valid_schemas:
```

Comment:
and here as well:
```
if self.validate_schemas:
```

Reply:
the problem here is that you need to let through the first query